src/Spotlight.php

```diff
@@ -82,14 +82,22 @@ public function execute(string $commandId, array $dependencies = []): void
     public function render(): View | Factory
     {
         return view('livewire-ui-spotlight::spotlight', [
-            'commands' => collect(self::$commands)->map(function (SpotlightCommand $command) {
-                return [
-                    'id' => $command->getId(),
-                    'name' => $command->getName(),
-                    'description' => $command->getDescription(),
-                    'dependencies' => $command->dependencies()?->toArray() ?? [],
-                ];
-            }),
+            'commands' => collect(self::$commands)
+                ->filter(function (SpotlightCommand $command) {
+                    if (! method_exists($command, 'shouldBeShown')) {
+                        return true;
+                    }
+
+                    return app()->call([$command, 'shouldBeShown']);
+                })
```

Comment:
```suggestion
                })
                ->values()
```

This could break because the array keys are changed. Adding `values()` will resolve this.